docs/stats/feldman_cousins.rst

```diff
@@ -0,0 +1,143 @@
+.. _feldman_cousins:
+
+Feldman and Cousins Confidence Intervals
+========================================
+Feldman and Cousins solved the problem on how to choose confidence intervals
+in a unified way (that is without basing the choice on the data)
+[Feldman1998]_. The functions ``gammapy.stats.fc_*`` give you access to a
+numerical solution to the Feldman Cousins algorithm. It can be used for any type
+of statistical distribution and is not limited to a Poisson process with
+background or a Gaussian with a boundary at the origin.
+
+The basic ingredient to `~gammapy.stats.fc_construct_acceptance_intervals_pdfs` is a matrix of
+:math:`P(X|\\mu)` (see e.g. equation (3.1) and (3.2) in [Feldman1998]_). Every row is a
+probability density function (PDF) of x and the columns are built up by varying
+the signal strength :math:`\\mu`. The other parameter is the desired confidence level
+(C.L.). The function will return another matrix of acceptance intervals where 1
+means the point is part of the acceptance interval and 0 means it is outside.
+This can be easily converted to upper and lower limits (`~gammapy.stats.fc_get_limits`),
+which simply connect the outside 1s for different :math:`\\mu`s. An upper or lower limit
+is obtained by drawing a vertical line at the measured value and calculating the
+intersection (`~gammapy.stats.fc_find_limit`).
```

Comment:
Here Sphinx gets confused and doesn't render the HTML output correctly (see [here](http://www.mpi-hd.mpg.de/personalhomes/deil/public/gammapy/issue_367/html/stats/feldman_cousins.html#feldman-and-cousins-confidence-intervals).

I think (but didn't verify) that the issue is the s you have directly following the math directive:
```
:math:`\\mu`s
```
Can you try putting this instead
```
:math:`\\mu` values
```
and then see if the link to 
```
`~gammapy.stats.fc_find_limit`
```
resolves correctly?